### PR TITLE
Give mommy more to say - Issue #32

### PR DESCRIPTION
**Pull request by isaacg1**
Fulfills Issue #32 by adding a bunch of new responses for chill mode.
https://github.com/Gankra/cargo-mommy/issues/32

(I've never submitted a pull request before, so please let me know if I should be doing something differently)

### CODE REVIEW

File: responses.json [hunk 1]
```diff
@@ -21,7 +27,13 @@
             "oh no did MOMMYS_ROLE's little AFFECTIONATE_TERM make a big mess~?",
             "MOMMYS_ROLE knows MOMMYS_PRONOUN little AFFECTIONATE_TERM can do better~",
             "MOMMYS_ROLE still loves you~",
-            "just a little further, sweetie~"
+            "just a little further, sweetie~",
+            "MOMMYS_ROLE thinks MOMMYS_PRONOUN little AFFECTIONATE_TERM is getting close~",
```
Comment (Gankra) [inline responses.json:31]:
```suggestion
            "MOMMYS_ROLE thinks MOMMYS_PRONOUN's little AFFECTIONATE_TERM is getting close~",
```

**Review by Gankra — APPROVED** [review]

**Comment by Gankra** [comment]
thanks for helping mommy~

**Comment by isaacg1** [comment]
I don't think this typo fix is correct - with the default settings, wouldn't it print "mommy thinks her's little girl is getting close~"?

**Comment by Gankra** [comment]
you are ABSOLUTELY correct lmao, sorry!

**Pull request merged**